resources/teacherTools/classDetails.jsp

```diff
@@ -25,6 +32,13 @@ try {
 catch (Exception e) {
 //	logger.error(e.getMessage());
 }
+
+HttpSession mySession = request.getSession();
+TeacherLogger tLogger = (TeacherLogger) mySession.getAttribute("tLogger");
+int teacherId = (int) mySession.getAttribute("teacherId");
+Connection conn = (Connection) mySession.getAttribute("conn");

```

Comment:
why do we need to Save the connection object here ? I guess we have the generic connection object atthe backend itself

Reply:
We do not need that.  I hae removed it.